main.go

```diff
@@ -194,9 +187,8 @@ func buildTomcatBundle() {
 		}
 	}
 	fmt.Println("\nSuccessfully re-packaged self-contained application:", filepath.Join("output", bundleName+"-application.zip"))
-	fmt.Println("To use it, simply unzip it like your usual Bonita Tomcat bundle, and run ./start-bonita[.sh|.bat]")
+	fmt.Println("\nTo use it, simply unzip it like your usual Bonita Tomcat bundle, and run ./start-bonita[.sh|.bat]")
```

Comment:
There is already a newline in Print**ln**

Reply:
To make it coherent with the Docker usage: the final message with the final output is distinctly visible in the stdout.

Reply:
ok, fine